### PR TITLE
Add WorkOS integration for Magic Link and Google

### PR DESCRIPTION
**Pull request by rjvani**

### CODE REVIEW

File: app/controllers/authorization_controller.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class AuthorizationController < ApplicationController
```
Comment (justintmckibben) [inline app/controllers/authorization_controller.rb:3]: we might want to test this

**Review by justintmckibben — APPROVED** [review]

**Review by jacoblb64 — APPROVED** [review]
This looks really dope man! Agreed with justin that we should have some tests / examples.

Is this a breaking change / what do we have to do to utilize this?

File: app/controllers/authorization_controller.rb (hunk 1 above)
Comment (rjvani) [reply app/controllers/authorization_controller.rb:3]: Added some tests. There's a stub for the Magic Link email, I'll add that in a subsequent PR.

**Comment by rjvani** [comment]
> This looks really dope man! Agreed with justin that we should have some tests / examples.
> 
> Is this a breaking change / what do we have to do to utilize this?

This isn't a breaking change. I've configured us with an API Key, account, and redirect URIs on our Google app. We just need to set up some environment variables on Heroku, and we should be good moving forward.

**Review by jacoblb64 — APPROVED** [review]

**Pull request merged**